docs/stdlib/v0/index.md

```diff
@@ -1,458 +0,0 @@
-# Protocol Documentation
```

Comment:
this is being deleted because it's correctly generated on the fly?

Reply:
Yeah!